### ISSUE TITLE
self.g_samples has gone through sigmoid normalization, so using sigmoid_cross_entropy_with_logits is not correct.

### ISSUE THREAD

**Issue by masonwang513** [ChengBinJin/V-GAN-tensorflow#16]
https://github.com/ChengBinJin/V-GAN-tensorflow/blob/00882dc6bda914c3acbd4b1b1c7c22be83a72bed/codes/model.py#L47

**Comment by ChengBinJin** [comment]
@masonwang513 For the discriminator, it is a binary classification problem isn't multi-classification. Therefore, I think it should be correct using sigmoid_cross_entropy_with_logits.
